tools/config.ts

```diff
@@ -29,44 +34,52 @@ export const TEST_DEST            = 'test';
 export const DOCS_DEST            = 'docs';
 export const APP_DEST             = `dist/${ENV}`;
 export const ASSETS_DEST          = `${APP_DEST}/assets`;
-export const BUNDLES_DEST         = `${APP_DEST}/bundles`;
 export const CSS_DEST             = `${APP_DEST}/css`;
-export const FONTS_DEST           = `${APP_DEST}/fonts`;
-export const LIB_DEST             = `${APP_DEST}/lib`;
+export const JS_DEST              = `${APP_DEST}/js`;
 export const APP_ROOT             = ENV === 'dev' ? `${APP_BASE}${APP_DEST}/` : `${APP_BASE}`;
 export const VERSION              = appVersion();
 
+export const CSS_PROD_BUNDLE      = 'all.css';
+export const JS_PROD_SHIMS_BUNDLE = 'shims.js';
+export const JS_PROD_APP_BUNDLE   = 'app.js';
+
 export const VERSION_NPM          = '2.14.2';
 export const VERSION_NODE         = '4.0.0';
 
-// Declare NPM dependencies (Note that globs should not be injected).
-export const NPM_DEPENDENCIES = [
-  { src: 'systemjs/dist/system-polyfills.js', dest: LIB_DEST },
-
-  { src: 'es6-shim/es6-shim.js', inject: 'shims', dest: LIB_DEST },
-  { src: 'reflect-metadata/Reflect.js', inject: 'shims', dest: LIB_DEST },
-  { src: 'systemjs/dist/system.src.js', inject: 'shims', dest: LIB_DEST },
-  { src: 'angular2/bundles/angular2-polyfills.js', inject: 'shims', dest: LIB_DEST },
-
-  // Faster dev page load
-  { src: 'rxjs/bundles/Rx.min.js', inject: 'libs', dest: LIB_DEST },
-  { src: 'angular2/bundles/angular2.js', inject: 'libs', dest: LIB_DEST },
-  { src: 'angular2/bundles/router.js', inject: 'libs', dest: LIB_DEST }, // use router.min.js with alpha47
-  { src: 'angular2/bundles/http.min.js', inject: 'libs', dest: LIB_DEST },
+interface InjectableDependency {
+  src: string;
+  inject: string | boolean;
+  dest?: string;
+}
 
-  { src: 'bootstrap/dist/css/bootstrap.min.css', inject: true, dest: CSS_DEST }
-];
+// Declare NPM dependencies (Note that globs should not be injected).
+export const DEV_NPM_DEPENDENCIES: InjectableDependency[] = normalizeDependencies([
+  { src: 'systemjs/dist/system-polyfills.src.js', inject: 'shims', dest: JS_DEST },
+  { src: 'reflect-metadata/Reflect.js', inject: 'shims', dest: JS_DEST },
+  { src: 'es6-shim/es6-shim.js', inject: 'shims', dest: JS_DEST },
+  { src: 'systemjs/dist/system.src.js', inject: 'shims', dest: JS_DEST },
+  { src: 'angular2/bundles/angular2-polyfills.js', inject: 'shims', dest: JS_DEST },
+  { src: 'rxjs/bundles/Rx.js', inject: 'libs', dest: JS_DEST },
+  { src: 'angular2/bundles/angular2.js', inject: 'libs', dest: JS_DEST },
+  { src: 'angular2/bundles/router.js', inject: 'libs', dest: JS_DEST },
+  { src: 'angular2/bundles/http.js', inject: 'libs', dest: JS_DEST },
+  { src: 'bootstrap/dist/css/bootstrap.css', inject: true, dest: CSS_DEST }
+]);
+
+export const PROD_NPM_DEPENDENCIES: InjectableDependency[] = normalizeDependencies([
```

Comment:
`angular2-polyfills.js` is missing here (zone not included, so change detection doesn't work in about page)